.github/workflows/pypi.yml

```diff
@@ -17,17 +17,32 @@ jobs:
           CIBW_SKIP: cp36-*
       - uses: actions/upload-artifact@v3
```

Comment:
We only need single `.tar.gz` file. If we add this steps after `pypa/cibuildwheel@v2.12.0` it will be executed and created per each `matrix.os`. It is not needed I guess, plus it will take more time.
But it will not be a problem, I can do it.